tests/framework/builder.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Define some helpers methods to create microvms from artifacts."""
+
+import json
+import os
+import uuid
+import host_tools.cargo_build as build_tools
+from conftest import init_microvm
+from framework.microvm import Microvm
+from framework.artifacts import Artifact, DiskArtifact
+
+
+class MicrovmBuilder:
+    """Build fresh microvms or restore from snapshot."""
+
+    def __init__(self, root_path, bin_cloner_path):
+        """Initialize microvm root and cloning binary."""
+        self.root_path = root_path
+        self.bin_cloner_path = bin_cloner_path
+
+    def build(self, kernel: Artifact, disks: [DiskArtifact], config: Artifact):
+        """Build and start a fresh microvm."""
```

Comment:
This comment might have slipped through - the method only builds the microvm.

Reply:
It also starts it in basic_config().